### PR TITLE
[nation3-passport-coop-with-delegations] Add minimum token holding to determine passport validity

### PR DESCRIPTION
**Pull request by johnmark13**
Changes proposed in this pull request:
- Adds the ERC20 token balance for the veNation linked to the passport used to determine if holding has fallen below 1.5 threshold required to vote.
- Fixes issue with Passport 0 no longer existing (on both N3 voting strategies)

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Everything else looks good and there are no open proposals 👍

File: src/strategies/nation3-passport-coop-with-delegations/index.ts [hunk 1]
```diff
@@ -1,9 +1,18 @@
-import { BigNumber } from '@ethersproject/bignumber';
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
 import { Multicaller } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
 
 export const author = 'nation3';
 export const version = '0.2.0';
```
[the remark this answers is not in the document]
Comment (johnmark13) [reply src/strategies/nation3-passport-coop-with-delegations/index.ts]: Done, apologies for missing it!

File: src/strategies/nation3-votes-with-delegations/index.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export async function strategy(
   const lastIndex = await erc721LastTokenIdCaller.execute();
   const lastTokenId = BigNumber.from(lastIndex.lastTokenId).toNumber();
 
-  for (let i = 0; i < lastTokenId; i++) {
+  for (let i = 1; i < lastTokenId; i++) {
```
[the remark this answers is not in the document]
Comment (johnmark13) [reply src/strategies/nation3-votes-with-delegations/index.ts:56]: I need to create two new PRs to fix the strategies completely, the versions on Master are both broken due to the non-existence of Token 0 which only occurred when I was creating this PR. Would you consider letting this one through, and I promise to create two separate PRs for the proper fixes?

**Review by ChaituVR — APPROVED** [review]

File: src/strategies/nation3-votes-with-delegations/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ChaituVR) [reply src/strategies/nation3-votes-with-delegations/index.ts:56]: Makes sense now, I overlooked at the change :D

**Pull request merged**